src/integrationTest/sam.test.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { mkdirpSync, readFileSync, removeSync } from 'fs-extra'
+import * as path from 'path'
+import * as vscode from 'vscode'
+import { SamLambdaRuntime } from '../../src/lambda/models/samLambdaRuntime'
+import { getSamCliContext } from '../../src/shared/sam/cli/samCliContext'
+import { runSamCliInit, SamCliInitArgs } from '../../src/shared/sam/cli/samCliInit'
+import { activateExtension, sleep, TIMEOUT } from './integrationTestsUtilities'
+
+const projectFolder = `${__dirname}`
+let projectSDK = ''
+let projectPath = ''
+let debuggerType = ''
+let documentUri: vscode.Uri
+
+const runtimes = [
+    { name: 'nodejs10.x', path: 'testProject/hello-world/app.js', debuggerType: 'node2' },
```

Comment:
What's `node2`?

Reply:
the name of the debugger, these come from VSCode and is just to make sure it's debugging the right langauge